### DISCUSSION TITLE
Generate entities with CLI: sqlite timestamps

### DISCUSSION THREAD

**Discussion by mainrs** [SeaQL/sea-orm#1687] [Q&A]
Is this even supported? My migration calls 

```rust
ColumnDef::new(FileRef::UpdatedAt)
                            .timestamp_with_time_zone()
                            .not_null(),
```

but this seems to be only Postgres. What is the right way to do this in SQLite? I want my entities to be auto-generated, if possible.

**Comment by billy1624** [comment]
Hey @mainrs, it's perfectly fine to define your column as `timestamp_with_time_zone()`. Only Postgres backend will have the timezone info. All other backend will interpret it as simple timestamp.

**Reply by mainrs** [reply]
That way, the auto-generated structs are of type `String` in my case. And not of type `chrono::DateTime` or any other datetime-like struct.

**Reply by billy1624** [reply]
Btw... why you need to re-generate the entity? You only need a single entity definition to run SeaORM on both Postgres and SQLite.

**Reply by mainrs** [reply]
> Btw... why you need to re-generate the entity? You only need a single entity definition to run SeaORM on both Postgres and SQLite.

I don't write the entities myself. I use the CLI to generate them. But it applies the wrong typing. `seaorm` supports `chrono` but it defines the database columns that are defined as timestamps to be a `String`. 

Even if I pass the CLI flag `--date-time-crate "chrono"`, it uses the `String` type for my `updated_at` column. My thought process was that, if I use the `timestamp_with_timezone()` builder method and generate an entity, it should be of type `chrono::DateTime` and not `String`.
